packages/carbon-graphs/CHANGELOG.md

```diff
@@ -6,3 +6,6 @@
 
 * Changed
   * Added a condition to check if graphContainer is present and then resize the graph.
+
+* Changed
+  * For consistency updated reflow in Graph and Gantt constructs to update the eventlines.
```

Comment:
Nit: sorry, we pushed another commit to main. Can you rebase again and move this line to the `unreleased` section?